cpp/src/parquet/metadata.cc

```diff
@@ -992,6 +1053,10 @@ std::shared_ptr<FileMetaData> FileMetaData::Subset(
   return impl_->Subset(row_groups);
 }
 
+std::string FileMetaData::SerializeUnencrypted(bool scrub, bool json) const {
+  return impl_->SerializeUnencrypted(scrub, json);
```

Comment:
Should we use `debug` instead of `json` here?

Reply:
Sent https://github.com/apache/arrow/pull/43375.